tests/sentry/api/endpoints/test_organization_details.py

```diff
@@ -783,24 +683,10 @@ def setUp(self):
         self.create_member(organization=self.organization, user=self.has_2fa, role="manager")
         assert Authenticator.objects.user_has_2fa(self.has_2fa)
 
-    @fixture
-    def path(self):
-        return reverse(
-            "sentry-api-0-organization-details", kwargs={"organization_slug": self.org_2fa.slug}
-        )
-
     def assert_2fa_email_equal(self, outbox, expected):
         assert len(outbox) == len(expected)
         assert sorted([email.to[0] for email in outbox]) == sorted(expected)
 
-    def assert_can_access_org_details(self, url):
```

Comment:
why were these 2 tests deleted?

Reply:
oh I think I see below where you just use `get_error_response` instead of calling these